lib/openers/pdf-view-opener.js

```diff
@@ -3,24 +3,34 @@
 import Opener from '../opener'
 import { isPdfFile } from '../werkzeug'
 
-function forwardSync (pdfView, texPath, lineNumber) {
-  if (pdfView != null && pdfView.forwardSync != null) {
-    pdfView.forwardSync(texPath, lineNumber)
-  }
+function sleep (time) {
+  return new Promise(resolve => setTimeout(resolve, time))
 }
 
 export default class PdfViewOpener extends Opener {
   async open (filePath, texPath, lineNumber) {
-    const openPaneItems = atom.workspace.getPaneItems()
-    for (const openPaneItem of openPaneItems) {
-      if (openPaneItem.filePath === filePath) {
-        forwardSync(openPaneItem, texPath, lineNumber)
-        return true
-      }
+    const texPane = atom.workspace.paneForURI(texPath)
+
+    // This prevents splitting the right pane multiple times
+    if (texPane) {
+      texPane.activate()
     }
 
-    // TODO: Make this configurable?
-    atom.workspace.open(filePath, {'split': 'right'}).then(pane => forwardSync(pane, texPath, lineNumber))
+    const options = {
+      searchAllPanes: true,
+      split: atom.config.get('latex.openerSplit')
+    }
+
+    const item = await atom.workspace.open(filePath, options)
+    const pane = atom.workspace.paneForItem(item)
+    if (pane && item && item.forwardSync) {
+      item.forwardSync(texPath, lineNumber)
+      // If we don't wait for pdf-view to do it's first update then it will just
+      // display a blank page when there are multiple jobs.
+      while (item.updating) {
+        await sleep(100)
```

Comment:
Looks like the issue is related to `Fit to width on open` setting in `pdf-view`. When that option is off everything works. When it is on then the resize fails when the tab is not visible. I'll remove the hack and investigate the fix for `pdf-view` or submit a bug report if I can't fix it.

Also, when not using DiCy activating the center pane before splitting isn't working because only the first job gets file name and line number for sync. It appears that this is because `showResult` is calling `getEditorDetails` on a `pdf-view`. I'll fix that in a separate PR since the fix will involve changes to `Composer`.